.github/workflows/aqua.yaml

```diff
@@ -11,20 +11,15 @@ jobs:
         run: printenv
       - name: mor 2
         run: cat /home/runner/work/_temp/_github_workflow/event.json
-      - name: install plugin
-        run: ./trivy plugin install github.com/Mor-s-organization/trivy-plugin-version
-
-      - name: Run Trivy vulnerability scanner in IaC mode
-        uses: aquasecurity/trivy-action@master
-        with:
-          scan-type: "fs"
-          security-checks: "vuln,config,secret"
-          hide-progress: false
-          format: "table"
+      - name: get trivy
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b . v0.34.0
+          ./trivy plugin install github.com/Mor-s-organization/trivy-plugin-version 
+          ./trivy fs --security-checks config,vuln,secret .
         env:
           AQUA_KEY: ${{ secrets.AQUA_KEY }}
           AQUA_SECRET: ${{ secrets.AQUA_SECRET }}
           TRIVY_RUN_AS_PLUGIN: "aqua"
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ github.token }}     
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0008   
**Check Name:** Unrestricted dependency version   
**Severity:** MEDIUM   
**Message:** Dependecy actions/checkout version v3 should be pinned to the commit sha  
[This comment was created by Aqua Pipeline]